linuxdoc/kernel_doc.py

```diff
@@ -435,6 +433,26 @@ def main():
             "Markup of the comments. Change this option only if you know"
             " what you do. New comments must be marked up with reST!"))
 
+    CLI.add_argument(
+        "--symbols-exported-method"
+        , default = DEFAULT_EXP_METHOD
+        , help    = (
+            "Indicate the way by which an exported symbol an exported symbol"
+            " is exported. Must be either 'macro' or 'attribute'."))
+
+    CLI.add_argument(
+        "--symbols-exported-identifiers"
+        , nargs   = "+"
+        , default = DEFAULT_EXP_IDS
+        , help    = "identifiers list that specifies an exported symbol")
+
+    CLI.add_argument(
+        "--known-attrs"
+        , default = ""
+        , nargs   = "+"
+        , help    = ("provides list of known attributes that has to be"
+                     " hidden from" " function prototypes"))
+
```

Comment:
Is this `" hidden from" " function prototypes"` a typo?

Reply:
Fixed